src/main/java/io/github/enkarin/bookcrossing/util/TimeUtil.java

```diff
@@ -0,0 +1,32 @@
+package io.github.enkarin.bookcrossing.util;
+
+import lombok.Setter;
+import lombok.experimental.UtilityClass;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+
+@UtilityClass
+public class TimeUtil {
+    @Setter
+    private static Clock clock = Clock.system(ZoneOffset.ofHours(7));
```

Comment:
Utility class is more universal because it can be used in other utility classes

Reply:
Is it worth doing a bean if it will lead to the loss of the possibility of using the date anywhere other than the components?

Reply:
> Is it worth doing a bean if it will lead to the loss of the possibility of using the date anywhere other than the components?

It's a matter of application design. Creating a clock bean forces you to explicitly provide clock as argument to all static methods. I think it's worth

Reply:
Done